ui/view.sh

```diff
@@ -25,38 +25,39 @@
 # Kopfzeile des Protokolls
 # --------------------------------------------------------------
 if [[ "${get[page]}" == "view" ]]; then
-	[ -f "${get_request}" ] && rm "${get_request}"
+	[ -f "${get_request:?}" ] && rm "${get_request}"
 
-	echo '
+cat <<EOF
```

Comment:
It might be that there is the need for an additional line break for each of this construct.

e.g.
```
cat <<EOF
            # add an additional blank line here
            <!-- Modal -->
```